### PR TITLE
Added http service, apiUrl in environment.ts and some changes for REA…

### PR DESCRIPTION
**Pull request by Setup1337**
…DME.md

### CODE REVIEW

**Review by ismv — CHANGES_REQUESTED** [review]
Fix the frontend warnings please

File: frontend/src/core/services/http.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
```
Comment (Anatolii-Ivanov) [inline frontend/src/core/services/http.service.spec.ts]: Drop this file, we won't write tests

**Comment by ihor-il** [comment]
You should try to build and lint your code locally before pushing it :|

File: frontend/src/core/services/http.service.spec.ts (hunk 1 above)
Comment (Setup1337) [reply frontend/src/core/services/http.service.spec.ts]: Done

File: frontend/src/core/services/http.service.spec.ts [hunk 2]
```diff
@@ -3,7 +3,6 @@ import { HttpService } from './http.service';
 describe('HttpService', () => {
```
Comment (ismv) [inline frontend/src/core/services/http.service.spec.ts]: you should delete this spec file

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by ismv — APPROVED** [review]

**Pull request merged**